courses/machine_learning/deepdive2/structured/solutions/4a_sample_babyweight.ipynb

```diff
@@ -0,0 +1,2818 @@
+{
```

Comment:
<p>I understand now why you wanted to use floats for train_percent and eval_percent, but there's still duplication of logic here. It can be fixed by casting lo and hi in create_data_split_sample_df.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/671/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>Done</p>